pkg/util/kafka/const.go

```diff
@@ -45,7 +52,13 @@ const (
 
 // used for Cruise Control configurations
 const (
```

Comment:
Question (for the team really, not just Darren): Don't really see why we have to separate const blocks here? Is it just for separation?

Reply:
This file was created by me - the intention was to create a visual separation between the Kafka server configurations and CC configurations keys.

Prior to this file, we used hard-coded strings for all these configuration keys everywhere, see https://github.com/banzaicloud/koperator/pull/917